components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/test/java/org/wso2/carbon/identity/configuration/mgt/core/util/TestUtils.java

```diff
@@ -0,0 +1,165 @@
+package org.wso2.carbon.identity.configuration.mgt.core.util;
```

Comment:
add license

Reply:
Fixed in d0a5b5ad8ff1bdab587b5bd0c8c0cc2c3b6d4afb